cmd/drive/main.go

```diff
@@ -2057,6 +2057,10 @@ func relativePathsOpt(root string, args []string, leastNonExistant bool) ([]stri
 			relPath = ""
 		}
 
+		// This function appears to be meant to return server-side (/-separated) paths.
+		// But its input is client side (maybe / or \) separted paths.
+		// filepath.ToSlash fixes that.
+		relPath = filepath.ToSlash(relPath)
```

Comment:
Did you mean to instead replace:
```diff
- relPath = "/" + relPath
+ relPath = filepath.ToSlash(relPath)
```
instead of keeping both?

Reply:
No. It's been a while so I forget the exact issue, but it was something like on Windows the input might be something like "photos\myalbum". Which should then be converted into "/photos/myalbum", but simply was being turned in to "/photos\myalbum". So both the ToSlash (convert "photos\myalbum" into "photos/myalbum") and the "/" prepend are necessary.
The command line was something like "drive pull photos/myalbum", it would then convert the slashes to one direction then the other ...